File: Semestrul 5/Proiectarea-Detaliata-a-Sistemelor-Software/Laborator/Daria/Laborator_10/decorator/SimplePizza.java
package Laborator.Daria.Laborator_10.decorator;

public class SimplePizza implements Pizza {
    @Override
    public String getDescription() {
        return "SimplePizza (ketchup, mozzarella)";
    }

    @Override
    public double getCost() {
        return 10.0;
    }
}

File: Semestrul 5/Proiectarea-Detaliata-a-Sistemelor-Software/Laborator/Paul/design-patterns/decorator/src/main/kotlin/pizza/SimplePizza.kt
package org.paul.pizza

class SimplePizza : Pizza {
    override fun getCost(): Double {
        return COST
    }

    override fun getDescription(): String {
        return "Simple Pizza (ketchup, mozzarella)"
    }

    companion object {
        private const val COST = 10.0
    }
}